client/src/databrowser/test/databrowser.unit.spec.js

```diff
@@ -724,6 +702,23 @@ tape('add unit to term', function (test) {
 	test.end()
 })
 
+tape('add additional attributes to term', function (test) {
+	test.timeoutAfter(100)
+	const tsv = [
+		`Level_1\tLevel_2\tVariable\ttype\tCategories\tadditional attributes`,
+		`Root\tTerm 2\tterm2\tinteger\t{\"999\":{\"label\":\"N/A:CCSS\"}}\t{"xx":"yy"}`
+	].join('\n')
+	try {
+		const results = parseDictionary(tsv)
+		const term = results.terms.find(i => i.id == 'term2')
+		test.ok(term, 'term is found by id=term2')
+		test.equal(term.xx, 'yy', 'new property found: term.xx=yy')
+	} catch (e) {
+		test.fail('additional attributes: ' + e)
+	}
+	test.end()
+})
+
 /***********************************
```

Comment:
here getHolder() shouldn't be needed. please assess if okay to delete @creilly8

Reply:
@xzhou82, Let me fix this in a different PR. It looks like at one point the parse functions directly reported errors in the viewport to the users. Now all the errors are throw statements. I'll go through and replace the outdated tests with the correct pass/fail test code.